src/roswire/ros2/node.py

```diff
@@ -40,10 +41,12 @@ def is_alive(self) -> bool:
             return True
         return False
 
-    def shutdown(self) -> None:
+    def shutdown(self, ignore_errors: bool = False) -> None:
         """Instructs this node to shutdown."""
```

Comment:
We should add a "Raises" section to this docstring that states this method produces a `NodeShutdownError` in the event of an error.